ui/src/progress_table_cell_type.test.tsx

```diff
@@ -19,7 +19,14 @@ import { ProgressTableCellType, XProgressTableCellType } from './progress_table_
 const
   name = 'progress-cell',
   progress = 0.0,
-  progressCellProps: ProgressTableCellType = { name }
+  progressCellProps: ProgressTableCellType = { name },
+  progressValues = [
+    {input: 0.88, output: '88%'},
+    {input: 0.888, output: '88.8%'},
+    {input: 0.8888, output: '88.88%'},
+    {input: 0.88888, output: '88.89%'},
+    {input: 0.88899, output: '88.9%'},
+    {input: 0.88999, output: '89%'},]  
```

Comment:
I would expect this to be:

```suggestion
    {input: 0.88899, output: '88.90%'},
    {input: 0.88999, output: '89.00%'},]  
```

The rationale being that if I have a list of floats, I want them all to have the same precision displayed (max 2 decimal places).

Reply:
@mturoci Thanks for the feedback and clarification. I had assumed that trailing 0's should be trimmed. But I can see now that you want progress data rendered with the same precision for consistency. I'll make the change and resubmit for review later today. Thanks!

Reply:
@mturoci I have a clarification question for you. Could you confirm that you do NOT expect to see the trailing zeros when the incoming percentage has less than 2 decimal places? In other words, the first 2 `progressValues` in the test case I added look like this:

```{typescript}
    {input: 0.88, output: '88%'},
    {input: 0.888, output: '88.8%'},
```

I assume this is what you want instead of seeing output such as `88.00%` and `88.80%` for these 2 examples? I assume that's the case since your comment was only on the last 2 items in the array, but I just wanted to double check before I commit something.

thanks!

Reply:
>  Could you confirm that you do NOT expect to see the trailing zeros when the incoming percentage has less than 2 decimal places?

Correct.

Reply:
@mturoci I have updated the PR with the change you requested. Here is a new screen shot from the Table example where you can see the trailing 0's for floats that are used in the example allow the data to be nicely aligned.

![output_example_issue_824_20240514](https://github.com/h2oai/wave/assets/61979454/4a093a92-2c2d-41d1-a75b-1a51820596f1)

I have also re-run all the unit tests and you can see they still pass:

![wave_ui_test_issue_824_20240514](https://github.com/h2oai/wave/assets/61979454/270de4ef-9185-48be-a8c3-e23342523019)

Let me know if you need any futher changes. Thanks!

Reply:
Perfect.